set.mm

```diff
@@ -98571,6 +98582,15 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
     UOPURANOZUSBNOZPURUSFGHZUPPVBUPUNVCUOVDUNGQEVCUEAGCQRSUOLTEVDUFBLDTRSUGUPVE
     ABUHUIURAUSBUJUKULUM $.
 
+  $( Equinumerosity of a disjoint union and a regular union of two disjoint
```

Comment:
```suggestion
  $( Equinumerosity of a disjoint union and a union of two disjoint
```

Reply:
OK, I guess plain `union` is better. For one thing, "regular" here wasn't supposed to be a reference to any form of regularity, it is just a non-technical word.

Perhaps the language here would be less awkward if there was a name for disjoint union which didn't have "union" in the name, but given how well-established disjoint union is as a term, that seems unlikely.

Will accept this suggestion.

Reply:
Hmm, I see I wasn't the only one to use "regular union" in this context: https://us.metamath.org/mpeuni/konigth.html does too. I guess I'll change that one as well.

Reply:
Yes, "disjoint union" is an example of a "red herring". (At first reading, I genuinely wondered what you meant by "regular union"... for at least five seconds!). And indeed, "regular" is one of the most overloaded terms in mathematics: for basically any noun, there is a slightly different "regular" notion.

If you're looking for another term, you can use "coproduct" !

Reply:
Yes, maybe "coproduct" would be a better name, see also Wikipedia https://en.wikipedia.org/wiki/Coproduct

Reply:
Well, the proposal was a bit of a joke, and tradition will make us stick to "disjoint union".  What we can do is add to the comment of ~df-dju :
```
The disjoint union, together with the left and right injections ~ df-inl and ~ df-inr , is
"the" coproduct in the category of sets and functions.
```
The main benefit of adding this sentence is that if someone searches for "coproduct" in the website, they will get this positive hit (and also, it links to the two definitions of the injections).

Reply:
(and we can even link to @avekens's proof that it is indeed the coproduct ~updjud !)

Reply:
I suppose but this sentence may baffle as many people as it helps. It refers to a lot of concepts which aren't treated in set.mm at all, or if they are in a very different section.

Even for people searching for "coproduct" I'm not sure they'd particularly be well served by landing on df-dju (since that's just one kind of coproduct).

Reply:
We have references to category theory in earlier places, see, for example, ~df-f1 :

>  Injections are precisely the monomorphisms in the category ` SetCat ` of sets and set functions, see ~ setcmon .

Reply:
it would be even better if we prove it's a coproduct, although I think the category theory development doesn't get far enough to define coproducts

Reply:
I mean to package that theorem into a definition which uses the word "coproduct" in the name. Otherwise it's difficult to look at that big logical formula and see "coproduct" from that.